### PR TITLE
Trus 754 - Declaration page

### PR DESCRIPTION
**Pull request by JoshuaH01**
Fixed failing tests after merging master

### CODE REVIEW

File: app/views/DeclarationView.scala.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        label = messages("declaration.lastName"),
+        ariaRequired = true
+        )
+
+        <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+                <span class="govuk-warning-text__assistive">@messages("site.warning")</span>
+                @messages("declaration.warning")
+            </strong>
```
[the remark this answers is not in the document]
Comment (JoshuaH01) [reply app/views/DeclarationView.scala.html]: This has been confirmed by Adam as a separate ticket

File: conf/app.routes [hunk 2]
```diff
@@ -368,4 +368,9 @@ POST       /agent-international-address                        controllers.Agent
 GET        /change-agent-international-address                  controllers.AgentInternationalAddressController.onPageLoad(mode: Mode = CheckMode)
 POST       /change-agent-international-address                  controllers.AgentInternationalAddressController.onSubmit(mode: Mode = CheckMode)
 
-GET        /confirmation                       controllers.ConfirmationController.onPageLoad()
+GET        /declaration                        controllers.DeclarationController.onPageLoad(mode: Mode = NormalMode)
+POST       /declaration                        controllers.DeclarationController.onSubmit(mode: Mode = NormalMode)
+GET        /change-declaration                  controllers.DeclarationController.onPageLoad(mode: Mode = CheckMode)
```
[the remark this answers is not in the document]
Comment (JoshuaH01) [reply conf/app.routes]: Removed the change routes and pushed up

File: conf/messages.en [hunk 3]
```diff
@@ -465,8 +466,8 @@ assetInterruptPage.subheading6 = Other assets
 assetInterruptPage.paragraph8 = You need a description and an estimate of the current value of the asset.
 
 
-agentAnswer.title = Check agency details
-agentAnswer.heading = Check agency details
+agentAnswer.title = Check agent details
```
[the remark this answers is not in the document]
Comment (JoshuaH01) [reply conf/messages.en]: As discussed with Adam messages are fine as they are

File: conf/messages.en [hunk 4]
```diff
@@ -739,14 +740,14 @@ individualBeneficiaryAddressUKYesNo.error.required = Select yes if the individua
...
-agentAddressYesNo.error.required = Select yes if the agency’s address is in the UK
+agentAddressYesNo.error.required = Select yes if agency’s address in the UK
 
 classBeneficiaryDescription.title = What is the description for the class of beneficiaries?
 classBeneficiaryDescription.heading = What is the description for the class of beneficiaries?
 classBeneficiaryDescription.checkYourAnswersLabel = What is the description for the class of beneficiaries?
 classBeneficiaryDescription.error.required = Enter the description for the class of beneficiaries
 classBeneficiaryDescription.error.length = The description of the class in the trust deed must be 56 characters or less
-classBeneficiaryDescription.error.invalid = The description of class in the trust deed must only include letters a to z, numbers, special characters, ampersands (&), backward slashes or spaces
+classBeneficiaryDescription.error.invalid = The description of class in the trust deed must only include letters a to z, numbers, spaces, special characters, ampersands (&), backword slashes, spaces
```
Comment (manishsw) [inline conf/messages.en]: @JoshuaH01  I think this is not required to be change. master is right.

**Pull request merged**